cmake/modules/platform/toolcfg/msvc.cmake

```diff
@@ -30,6 +30,7 @@ list(APPEND OMR_PLATFORM_COMPILE_OPTIONS
 	/Zm400  # Precompiled header memory allocation limit
 	/wd4577 # Disable warning: Specifying noexcept when exceptions are disabled
 	/wd4091 # Disable warning: Caused by broken windows SDK, see also https://connect.microsoft.com/VisualStudio/feedback/details/1302025/warning-c4091-in-sdk-7-1a-shlobj-h-1051-dbghelp-h-1054-3056
+	/wd4200 # Disable warning: FAM in struct/union is valid in C and widely supported in C++
```

Comment:
This option must also be added in `omrmakefiles/rules.win.mk`:
```
ifeq ($(OMR_ENHANCED_WARNINGS),1)
    # -wd4200: Disable warnings regarding flexible array members.
    GLOBAL_CFLAGS   += -W3 -wd4200
    GLOBAL_CXXFLAGS += -W3 -wd4200
    GLOBAL_ASFLAGS  += -W3 -wd4200
endif
```

Reply:
hm, I don't understand why, or the difference between /Wx and -Wx, but I'll take your word for it.

Reply:
MSVC accepts options beginning with a dash `-` or a slash `/` and handles them equivalently.
I suggested using `-` here for consistency with how other options are expressed, but there's no functional difference.